File: CRUD/PHP/Ajax V3/entities/Usuario.php
<?php

class Usuario {

  private $id;
  private $nombre;
  private $clave;
  private $id_tipo;
  private $fecha_registro;

  private $tipo;

  public function __construct(){
    $this->id = "";
		$this->nombre = "";
		$this->clave = "";
		$this->id_tipo = "";
		$this->fecha_registro = "";

    $this->tipo = "";
   }

  public static function CreateUsuario($id,$nombre,$clave,$id_tipo,$fecha_registro) {
   		$instance = new self();
    	$instance->id = $id;
  		$instance->nombre = $nombre;
  		$instance->clave = $clave;
  		$instance->id_tipo = $id_tipo;
  		$instance->fecha_registro = $fecha_registro;
    	return $instance;
  }

  public function getId() {
    	return $this->id;
  }

  public function setId($id) {
    	$this->id = $id;
  }
  
  public function getNombre() {
   	 	return $this->nombre;
  }

  public function setNombre($nombre) {
    	$this->nombre = $nombre;
  }
  
  public function getClave() {
    	return $this->clave;
  }

  public function setClave($clave) {
    	$this->clave = $clave;
  }

  public function getId_tipo() {
   	 	return $this->id_tipo;
  }

  public function setId_tipo($id_tipo) {
   	 	$this->id_tipo = $id_tipo;
  }
  
  public function getTipo() {
      return $this->tipo;
  }

  public function setTipo($tipo) {
      $this->tipo = $tipo;
  }

  public function getFecha_registro() {
    	return $this->fecha_registro;
  }

  public function setFecha_registro($fecha_registro) {
    	$this->fecha_registro = $fecha_registro;
  }
  
  public function __destruct(){
    	$this->id = "";
		$this->nombre = "";
		$this->clave = "";
		$this->id_tipo = "";
		$this->fecha_registro = "";
   }  
   
   public function toString() {
		return "Usuario{" . "id=" . $this->id . ", nombre=" . $this->nombre . ", clave=" . $this->clave . ", id_tipo=" . $this->id_tipo . ", fecha_registro=" . $this->fecha_registro . '}';
   }
  
}

?>

File: CRUD/Java usando NetBeans/Aplicacion Web usando MVC Spring con Hibernate/SistemaWebSpringHibernate/SistemaWebSpringHibernate/src/main/java/sistema/models/Usuario.java
// Written by Ismael Heredia in the year 2016

package sistema.models;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "usuarios")
public class Usuario {

    @Id
    @Column(name = "id_usuario")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    int id_usuario;
    @Column(name = "usuario")
    String nombre;
    @Column(name = "clave")
    String password;
    @Column(name = "tipo")
    int tipo;
    @Column(name = "fecha_registro")
    String fecha_registro;

    public int getId_usuario() {
        return id_usuario;
    }

    public void setId_usuario(int id_usuario) {
        this.id_usuario = id_usuario;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public int getTipo() {
        return tipo;
    }

    public void setTipo(int tipo) {
        this.tipo = tipo;
    }

    public String getFecha_registro() {
        return fecha_registro;
    }

    public void setFecha_registro(String fecha_registro) {
        this.fecha_registro = fecha_registro;
    }

    public Usuario(int id_usuario, String nombre, String clave, int tipo, String fecha_registro) {
        this.id_usuario = id_usuario;
        this.nombre = nombre;
        this.password = password;
        this.tipo = tipo;
        this.fecha_registro = fecha_registro;
    }
    
    public Usuario() {
        
    }

    @Override
    public String toString() {
        return "Usuario{" + "id_usuario=" + id_usuario + ", nombre=" + nombre + ", password=" + password + ", tipo=" + tipo + ", fecha_registro=" + fecha_registro + '}';
    }
    
}
